pkgs/development/compilers/zig/hook.nix

```diff
@@ -11,12 +11,13 @@ makeSetupHook {
   substitutions = {
     zig_default_flags =
       let
-        cpu = "-Dcpu=baseline";
-        releaseType = if lib.versionAtLeast zig.version "0.11"
-                      then "-Doptimize=ReleaseSafe"
-                      else "-Drelease-safe=true";
+        releaseType =
+          if lib.versionAtLeast zig.version "0.11" then
+            "-Doptimize=ReleaseSafe"
+          else
+            "-Drelease-safe=true";
```

Comment:
More lines, same or less clarity. Looks nice!

Reply:
Why the saltyness?

This conforms to the (draft) RFC 101
- https://github.com/happysalada/rfcs/blob/nix_formatting/0101-nix%20formatting.md#if-then-else
- > Two spaces are used for each indentation level, there is no vertical alignment

I know this is not implemented yet, but this was the original formatting, I don't see a reason to change the formatting in the opposite direction for no apparent gains.